nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -184,4 +184,27 @@ public void externalInitSupport() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void generatedAsUnannotated() {
+    compilationHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:TreatGeneratedAsUnannotated=true"))
+        .addSourceLines(
```

Comment:
Is this better than just doing a small test file and doing `addSourceFile`?

Reply:
I've come to like this method for local readability of the test file.  Before I was getting lazy and adding more and more stuff to huge resource files rather than writing clean separate tests